### PR TITLE
Update AdService Dependencies

### PR DESCRIPTION
**Pull request by austinlparker**
Part of #919

### CODE REVIEW

File: src/adservice/build.gradle [hunk 1]
```diff
@@ -47,11 +48,11 @@ dependencies {
                 "io.opentelemetry:opentelemetry-api",
                 "io.opentelemetry:opentelemetry-sdk",
                 "io.opentelemetry.instrumentation:opentelemetry-instrumentation-annotations",
-                "org.apache.logging.log4j:log4j-core:2.17.2"
+                "org.apache.logging.log4j:log4j-core:3.0.0-alpha1"
```
Comment (trask) [inline src/adservice/build.gradle:51]: I'd recommend against updating to `-alpha` version of log4j

**Review by julianocosta89 — APPROVED** [review]
LGTM

**Comment by julianocosta89** [comment]
Please add a changelog

**Comment by austinlparker** [comment]
Will add changelogs for all of the dependency updates when we cut a release

**Pull request merged**